### PR TITLE
add option to specify meta and master resource names

### PR DESCRIPTION
**Pull request by assumednormal**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -784,9 +802,17 @@ func (c *Client) establishRegion(originalReg *regioninfo.Info, host string, port
 			}
 		}
 		if c.clientType == AdminClient {
-			host, port, err = c.zkLookup(ctx, zk.Master)
+			if c.master == (zk.ResourceName)("") {
```
Comment (ashishgandhi) [inline client.go]: You could initialize `c.master` and `c.meta` in the constructor (https://github.com/assumednormal/gohbase/blob/master/client.go#L226) if they are empty.

Comment (assumednormal) [inline client.go]: Done.

Comment (ashishgandhi) [inline client.go]: Then you don't need a test here, right? After you test this in the setter funcs. :smile:

**Comment by ashishgandhi** [comment]
:+1:

**Pull request merged**